### PR TITLE
Improve performance - Fixing issue #563

### PR DESCRIPTION
**Pull request by fabrianibrahim**
Improve performance for month view on react native

### CODE REVIEW

File: src/components/CalendarBodyForMonthView.tsx [hunk 1]
```diff
@@ -172,14 +172,6 @@ function _CalendarBodyForMonthView<T extends ICalendarEventBase>({
...
                 {date &&
                   events
-                    .sort((a, b) => {
-                      if (dayjs(a.start).isSame(b.start, 'day')) {
-                        const aDuration = dayjs.duration(dayjs(a.end).diff(dayjs(a.start))).days()
-                        const bDuration = dayjs.duration(dayjs(b.end).diff(dayjs(b.start))).days()
-                        return bDuration - aDuration
-                      }
-                      return a.start.getTime() - b.start.getTime()
-                    })
```
[the remark this answers is not in the document]
Comment (fabrianibrahim) [reply src/components/CalendarBodyForMonthView.tsx:182]: thanks for the quick review and reply. good idea, I will add the props :)

[the remark this answers is not in the document]
Comment (fabrianibrahim) [reply src/components/CalendarBodyForMonthView.tsx:182]: done. please review it again
Screen Shot 2022-07-15 at 11 00 09 am

[the remark this answers is not in the document]
Comment (acro5piano) [reply src/components/CalendarBodyForMonthView.tsx:182]: Nice!

**Comment by acro5piano** [comment]
Thanks for your contribution! I'll add doc for this prop later and release it soon.

**Comment by acro5piano** [comment]
Oh sorry you've already added the document - it is ready to release!

**Pull request merged**